src/test/java/doke/TodoTest.java

```diff
@@ -0,0 +1,20 @@
+package doke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TodoTest {
+
+    @Test
+    public void getType_emptyString_stringTReturned() {
```

Comment:
I like the naming convention of these test cases - they are very clear in their domain and expected outcomes